src/insertItem.ts

```diff
@@ -0,0 +1,482 @@
+    items.push(getQuickPickItem("Cosmos DB SQL Container"));
+    items.push(getQuickPickItem("Cosmos DB Gremlin Graph"));
+    items.push(getQuickPickItem("Cosmos DB Table Storage Table"));
+    items.push(getQuickPickItem("Data Lake Store Account"));
+    items.push(getQuickPickItem("DNS Record"));
+    items.push(getQuickPickItem("DNS Zone"));
+    items.push(getQuickPickItem("Function"));
+    items.push(getQuickPickItem("KeyVault"));
+    items.push(getQuickPickItem("KeyVault Secret"));
+    items.push(getQuickPickItem("Kubernetes Service Cluster"));
+    items.push(getQuickPickItem("Linux VM Custom Script"));
+    items.push(getQuickPickItem("Load Balancer External"));
+    items.push(getQuickPickItem("Load Balancer Internal"));
+    items.push(getQuickPickItem("Log Analytics Solution"));
+    items.push(getQuickPickItem("Log Analytics Workspace"));
+    items.push(getQuickPickItem("Logic App"));
+    items.push(getQuickPickItem("Logic App Connector"));
+    items.push(getQuickPickItem("Managed Identity (User Assigned)"));
+    items.push(getQuickPickItem("Media Services"));
+    items.push(getQuickPickItem("MySQL Database"));
+    items.push(getQuickPickItem("Network Interface"));
+    items.push(getQuickPickItem("Network Security Group"));
+    items.push(getQuickPickItem("Network Security Group Rule"));
+    items.push(getQuickPickItem("Public IP Address"));
+    items.push(getQuickPickItem("Public IP Prefix"));
+    items.push(getQuickPickItem("Recovery Service Vault"));
+    items.push(getQuickPickItem("Redis Cache"));
+    items.push(getQuickPickItem("Route Table"));
+    items.push(getQuickPickItem("Route Table Route"));
+    items.push(getQuickPickItem("SQL Database"));
+    items.push(getQuickPickItem("SQL Database Import"));
+    items.push(getQuickPickItem("SQL Server"));
+    items.push(getQuickPickItem("Storage Account"));
+    items.push(getQuickPickItem("Traffic Manager Profile"));
+    items.push(getQuickPickItem("Ubuntu Virtual Machine"));
+    items.push(getQuickPickItem("Virtual Network"));
+    items.push(getQuickPickItem("VPN Local Network Gateway"));
+    items.push(getQuickPickItem("VPN Virtual Network Gateway"));
+    items.push(getQuickPickItem("VPN Virtual Network Connection"));
+    items.push(getQuickPickItem("Web App"));
+    items.push(getQuickPickItem("Web Deploy for Web App"));
+    items.push(getQuickPickItem("Windows Virtual Machine"));
+    items.push(getQuickPickItem("Windows VM Custom Script"));
+    items.push(getQuickPickItem("Windows VM Diagnostics Extension"));
+    items.push(getQuickPickItem("Windows VM DSC PowerShell Script"));
+    return items;
+}
+
+export function getQuickPickItem(label: string): vscode.QuickPickItem {
+    return { label: label };
+}
+export function getInsertItemType(): QuickPickItem<SortType>[] {
+    let items: QuickPickItem<SortType>[] = [];
+    items.push(new QuickPickItem<SortType>("Function", SortType.Functions, "Insert a function"));
```

Comment:
Need to be consistent with "inserts" vs "insert" (either way is fine)

Reply:
Fixed.

Reply:
Fixed.